src/clientMods.ts

```
@@ -0,0 +1,528 @@
+      for (const urlTemplate of urls) {
+        const modNameOnly = mod.name.split('.').pop()
+        const modFolder = repo.prefix === false ? modNameOnly : typeof repo.prefix === 'string' ? `${repo.prefix}/${modNameOnly}` : mod.name
+        const url = new URL(`${modFolder}/${urlTemplate}`, normalizeRepoUrl(repo.url).replace(/\/$/, '') + '/').href
+        // eslint-disable-next-line no-useless-catch
+        try {
+          const response = await fetch(url)
+          if (!response.ok) throw new Error(`Failed to fetch ${url}: ${response.status} ${response.statusText}`)
+          return await response.text()
+        } catch (e) {
+          // errored.push(String(e))
+          throw e
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)